pkg/gitproviders/provider.go

```diff
@@ -511,6 +515,10 @@ func (p defaultGitProvider) GetCommitsFromOrgRepo(orgRepRef gitprovider.OrgRepos
 	// currently locking the commit list at 10. May discuss pagination options later.
 	commits, err := ur.Commits().ListPage(ctx, targetBranch, pageSize, pageToken)
 	if err != nil {
+		if strings.Contains(err.Error(), "409 Git Repository is empty") {
```

Comment:
Why is this block repeated?

Reply:
One is for org repo and one is for use repo. I guess I can combine them.

Reply:
One way to avoid repeating would be to do something like this:

```suggestion
		if isEmptyRepoError(err) {
```